mimic/rest/maas_api.py

```diff
@@ -620,16 +622,25 @@ def service_json_home(self, request, tenant_id):
     @app.route('/v1.0/<string:tenant_id>/views/agent_host_info', methods=['GET'])
     def view_agent_host_info(self, request, tenant_id):
         """
-        No agent monitoring. For now, always return 400.
+        Return 400 until the fifth attempt, then start to return data as if the
+        Agent is truly installed and working.
         """
-        request.setResponseCode(400)
-        return json.dumps({
-            "type": "agentDoesNotExist",
-            "code": 400,
-            "message": "Agent does not exist",
-            "details": "Agent XYZ does not exist.",
-            "txnId": ".fake.mimic.transaction.id.c-1111111.ts-123444444.v-12344frf"
-        })
+        entity_id = request.args['entityId'][0].strip()
```

Comment:
what if the entityId is not provided? The MAAS api returns a 400. Can we also include that behavior.